install-deps.sh

```diff
@@ -302,6 +315,18 @@ if [ x$(uname)x = xFreeBSDx ]; then
 
     exit
 else
+    while [ $# -ge 1 ]; do
+    case $1 in
+        --crimson)
+            ceph_osd=crimson-osd
```

Comment:
we don't use ceph_osd anywhere in install-deps, can you help me understand why do we need this? Perhaps did you intend to add 
```
--crimson)
            WITH_SEASTAR=1
```
for install-deps?